hw14/src/Main.java

```diff
@@ -0,0 +1,59 @@
+import java.util.Random;
+import java.util.Scanner;
+
+public class Main {
+    public static void main(String[] args){
+        Main main = new Main();
+        main.playRandomWordGame();
+    }
+
+    private void playRandomWordGame(){
+        String[] words = {"apple", "orange", "lemon",
+                "banana", "apricot", "avocado" ,
+                "broccoli", "carrot", "cherry",
+                "garlic", "grape", "melon",
+                "leak", "kiwi", "mango",
+                "mushroom", "nut", "olive",
+                "pea", "peanut", "pear",
+                "pepper", "pineapple", "pumpkin", "potato"};
+        final int MAX_SYMBOL_COUNT = 15;
+
+        long seed = System.currentTimeMillis();
+        Random rnd = new Random(seed);
+        Scanner scanner = new Scanner(System.in);
+        String userWord;
+        char[] workInProgress = new char[MAX_SYMBOL_COUNT];
+
+        String word = words[Math.abs(rnd.nextInt()) % words.length];
+        System.out.println("I thought of a word, you need to guess it!");
+
+        do{
+            for (int i = 0; i < MAX_SYMBOL_COUNT; i++){
+                if (i < word.length()) {
+                    char wordCh = word.charAt(i);
+                    char wipCh = workInProgress[i];
+                    if (wordCh == wipCh)
+                        System.out.print(wordCh);
+                    else
+                        System.out.print('#');
+                }
+                else{
+                    System.out.print('#');
+                }
+            }
+
+            System.out.println();
+            System.out.print("Input your word: ");
+            userWord = scanner.nextLine();
+
+            for (int i = 0; i < word.length() && i < userWord.length(); i++){
```

Comment:
Would be better to use i<Math.min(word.len(), userWord.len()) here. -1 score

Reply:
Or just word.length() as you are using its length above in line 32

Reply:
processed.